### PR TITLE
challenge-programs: Add FAQ for the English version

### PR DESCRIPTION
**Pull request by BellaXiang**
Add FAQ

### CODE REVIEW

File: challenge-Programs/performance-challenge.md [hunk 1]
```diff
@@ -91,3 +93,22 @@ How to exchange: Fill out the Redeem Gifts form after the end of this season (Th
...
 **Good luck and look forward to your brilliant idea!!**
 
+## FAQ (Updating)
+
+### Q1: How to change the registration information? How about the change of the Contribution Credit after the registration information changes?
+If you want to change the signup information, you need to close the wrong signup Issue and open a new one for signing up. Editing on the original Issue is unavailable for the moment. There are three kinds of changes related to the Contribution Credit after the change of registration information:
+
+- The total Contribution Credit owned by a team will not be affected by the change of team members;
+- If you join a team, your individual Contribution Credit will remain at the state before joining the team, and all the Contribution Credit you gained after joining the team will be owned by the team;
+- If you quit a team, your original Contribution Credit is "0" because you can't inherit the team Contribution Credit.
```
Comment (dcalvin) [inline challenge-Programs/performance-challenge.md]: What if the has credits before joining the team and now he quits to be individual again? He should still have the original credit according to point 1. @zz-jason

File: challenge-Programs/performance-challenge.md [hunk 2]
```diff
@@ -91,3 +93,22 @@ How to exchange: Fill out the Redeem Gifts form after the end of this season (Th
...
+
+### Q2: How to give up/change the picked Issue?
+
+- Give up a Issue: commenting "give-up-challenge" on the picked Issue
+- Change a Issue: giving up the Issue first, then picking up a new Issue
+
+### Q3: How does the previous contributions in the community combine with the Performance Challenge Program?
+Performance Challenge Program is a brand new community activity embracing TiDB & TiKV open source community. While the previous contributions in the community can't be converted into the Contribution Credit, the participants who are Active Contributors and the superior role in the commnity can directly pick up the "Medium" and "Hard" Issues. (See [here](https://pingcap.com/community-cn/organization/) for more details about the roles in the commnity )
```
Comment (dcalvin) [inline challenge-Programs/performance-challenge.md]:
```suggestion
Performance Challenge Program is a brand new community activity that embraces TiDB & TiKV open source community. While the previous contributions in the community can't be converted into the Contribution Credit, the participants who are Active Contributors or above in the commnity can directly pick up the "Medium" or "Hard" Issues. (See [here](https://pingcap.com/community-cn/organization/) for more details about the roles in the commnity).
```

File: challenge-Programs/performance-challenge.md (hunk 1 above)
Comment (BellaXiang) [reply challenge-Programs/performance-challenge.md]: Yes, "your original Contribution Credit is '0' " will be deleted. Thank you, Calvin!

File: challenge-Programs/performance-challenge.md [hunk 3]
```diff
@@ -16,58 +16,60 @@ We can’t wait to see innovative ideas sparkled! Participants are welcomed to j
...
-When your solution is ready, you can open a PR and commit the code to the corresponding GitHub Repo (i.e.,tidb) as a GitHub Pull Request (aka PR). Remember to @ the mentor responsible of this Issue to review your PR in the comment after the PR is submitted. The Mentor will check your PR as soon as possible.
+When your solution is ready, commit the code and submit a PR and to the corresponding GitHub Repo (i.e., tidb). Remember to request the review from the mentor responsible for this issue. The mentor will check your PR as soon as possible.
 
-How to submit: When you finish the coding, you can submit the GitHub Pull Request (PR) to the corresponding Repo. About how to submit PR, see [GitHub Workflow](https://github.com/pingcap/community/blob/master/contributors/workflow.md). There is also [SRE-BOT Command](https://github.com/pingcap/community/blob/master/contributors/command-help.md) Help for your reference.
+How to submit:  see [GitHub Workflow](https://github.com/pingcap/community/blob/master/contributors/workflow.md). There is also [SRE-BOT Command](https://github.com/pingcap/community/blob/master/contributors/command-help.md) Help for your reference.
```
Comment (Soline324) [inline challenge-Programs/performance-challenge.md]: * Seems like there is an extra space between the words "submit" and "see"
* The text of the link (https://github.com/pingcap/community/blob/master/contributors/command-help.md) is "SRE-BOT Command Help"

@BellaXiang @dcalvin PTAL

**Pull request merged**